README.md

```diff
@@ -1,3 +1,43 @@
-# ajax-project-template
+# City Quality of Life
 
 A dynamic HTML, CSS, and JavaScript solo project.
+
+Live Link: https://cbrisk.github.io/ajax-project/
+
+## Technologies Used
+
+- HTML5
+- CSS3
+- JavaScript
+
+## Main Features of this App
+
+* Users can view quality of life ratings for a couple of preselected cities
+* Users can also enter any city they choose and see whether there is available data for that city
+* Users can view ratings for available cities along with a summary
+* Users can save cities in a favorites list and view that list
+* Users can delete cities in the favorites list
+* Users can compare the ratings of two cities (only select cities)
+* Users can view the compared cities ratings with color coding on higher scores
+
+## Preview
+
+![SGT React](images/preview.gif)
```

Comment:
The square brackets hold alt text like img tags do, please change that to `Preview image of City Quality of Life`